include/arch/arm/aarch32/arch.h

```diff
@@ -183,10 +183,10 @@ extern "C" {
 /* Legacy case: retain containing extern "C" with C++ */
 #ifdef CONFIG_ARM_MPU
 #ifdef CONFIG_CPU_HAS_ARM_MPU
-#include <arch/arm/aarch32/cortex_m/mpu/arm_mpu.h>
+#include <arch/arm/aarch32/mpu/arm_mpu.h>
```

Comment:
I think it's necessary for you to squish this into previous commit so that the tree is still bisectable.

Reply:
I left a compatibility layer in include/arch/arm/aarch32/cortex_m/mpu that includes the header files from the new location.  I went back and did builds after the 1st commit, after the second commit, and then after the 3rd to make sure they still built.  Unless I missed something, it should still be bisectable.

Reply:
oh, missed that. thanks!